<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>30</y>
      <width>501</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QGroupBox{
	border: 1px solid; 
	border-radius: 10px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>10</y>
       <width>500</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Каноническое уравнение</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>40</y>
       <width>500</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Параметрическое уравнение</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>70</y>
       <width>500</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Алгоритм средней точки</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_4">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>100</y>
       <width>500</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Алгоритм Брезенхема</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_6">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>130</y>
       <width>500</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Библиотечная функция</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>2</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Алгоритмы построения</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>1400</width>
      <height>1080</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>235</y>
      <width>500</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QGroupBox{
	border: 1px solid; 
	border-radius: 10px;
}
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="widget_back" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_back_color">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgb(215,215,215);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgb(205,205,205);
    border-style: inset;
}</string>
     </property>
     <property name="text">
      <string>Поменять</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>325</y>
      <width>500</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QGroupBox{
	border: 1px solid; 
	border-radius: 10px;
}
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="widget_line" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_line_color">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgb(215,215,215);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgb(205,205,205);
    border-style: inset;
}</string>
     </property>
     <property name="text">
      <string>Поменять</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>202</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Цвет фона:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Цвет линии:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>500</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QGroupBox{
	border: 1px solid; 
	border-radius: 10px;
}
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QDoubleSpinBox" name="y_center">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>150</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1080.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="x_center">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>150</width>
       <height>32</height>
      </rect>
     </property>
     <property name="maximum">
      <double>1400.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Yц:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Xц:</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Координаты центра:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>500</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QGroupBox{
	border: 1px solid; 
	border-radius: 10px;
}
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QDoubleSpinBox" name="radius_y">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>150</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <double>1080.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="radius_x">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>150</width>
       <height>32</height>
      </rect>
     </property>
     <property name="maximum">
      <double>1400.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ry:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Rx:</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Радиус окружности:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_draw_circle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(215,215,215);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgb(205,205,205);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>Построить окружность</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>680</y>
      <width>500</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QGroupBox{
	border: 1px solid; 
	border-radius: 10px;
}
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QDoubleSpinBox" name="number">
     <property name="geometry">
      <rect>
       <x>329</x>
       <y>10</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>540.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="step">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>359.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Количество:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Шаг:</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_drawspectre_circle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>740</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(215,215,215);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgb(205,205,205);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>Построить спектр окружностей</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_compare_time_circle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>820</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(215,215,215);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgb(205,205,205);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>Сравнение времени построения окружностей</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_compare_time_ellipse">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>890</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(215,215,215);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgb(205,205,205);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>Сравнение времени построения эллипсов</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clearscreen">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>980</y>
      <width>280</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(215,215,215);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgb(205,205,205);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>Очистить экран</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Радиусы эллипса:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="radius">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>480</y>
      <width>150</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1080.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_draw_ellipse">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>620</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(215,215,215);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgb(205,205,205);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>Построить эллипс</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_drawspectre_ellipse">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>740</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(215,215,215);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgb(205,205,205);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>Построить спектр эллипсов</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
